dev-docs/publisher-api-reference/triggerBilling.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: api_prebidjs
+title: pbjs.triggerBilling
+description:
+sidebarType: 1
+---
+
+
+Allows a publisher the option to manually trigger billing for a winning bid (The winning bid to be billed is passed through the function below as an argument).
+
+{% highlight js %}
+
+pbjs.triggerBilling({adId});
```

Comment:
@jlquaccia - this doesn't look like valid JSON to me. Or did you intend this to be 

```
pbjs.triggerBilling(ADID_OF_RENDERED_AD);
```

The curly braces make this look like JSON but maybe that's not the intention?

Reply:
Thanks @bretg.  Good call, agree that could've been made more clear.  Pushed a commit just now updating that example to illustrate that the winning bid to be billed should be passed through as the arg for that function.